components/posts/data.js

```diff
@@ -0,0 +1,32 @@
+const bearer = "joDiolo3-TL5MGpDG2TVAOSn-EN5Z3G3qXYpKuCg-l0";
+
+export async function getPhPosts() {
+  const params = {
+    headers: {
+      Authorization: "Bearer " + bearer
```

Comment:
```suggestion
      Authorization: `Bearer ${bearer}`
```

I think it's cleaner to write it using this

Reply:
Cool thank you!

Done, resolving